core/plugins/playwright/src/lib/plugins-scully-plugin-playwright.ts

```diff
@@ -0,0 +1,188 @@
+import { createFolderFor, HandledRoute, logError, logWarn, registerPlugin, scullyConfig, yellow, cliOptions } from '@scullyio/scully';
+import { readFileSync } from 'fs';
+import { jsonc } from 'jsonc';
+import { join } from 'path';
+import { Browser, Page } from 'playwright';
+import { launchedBrowser, reLaunch, waitForIt } from './plugins-scully-plugin-playwright-utils.js';
+const { showBrowser, ssl } = cliOptions;
+
+let version = '0.0.0';
+try {
+  const pkg = join(__dirname, '../../../package.json');
```

Comment:
Is this still the correct depth? This thing should be reading the package.json from scully itself, right?